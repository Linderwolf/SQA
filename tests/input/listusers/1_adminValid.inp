admin
list_users
logout