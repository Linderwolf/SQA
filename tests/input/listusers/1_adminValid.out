Welcome to Vapour!
Please enter your Username to login:
Welcome admin! Plese enter a transaction:
Name            Type Credit
admin           AA 000000.00
fullstandard    FS 000100.00
buystandard     BS 001000.00
sellstandard    SS 000000.00
JohnDoe         BS 000000.00